dash/dash.py

```diff
@@ -36,6 +36,7 @@
 from . import _watch
 from ._utils import get_asset_path as _get_asset_path
 from ._utils import create_callback_id as _create_callback_id
+from ._utils import get_relative_path as _get_relative_path
```

Comment:
Does this pattern (`import x as _x`) serve a purpose, given that we re-export the symbols we're interested in from `__init__.py`?

Reply:
I suppose a user could code complete their way into `dash.dash`, instead of using `dash.Dash`:
![image](https://user-images.githubusercontent.com/1280389/72016410-bfbf6800-3231-11ea-95df-e74058da9d7f.png)

But we've got a bunch of other extraneous stuff in there already like `dash_renderer`